crates/types/src/net/ingest.rs

```diff
@@ -13,15 +13,19 @@ use std::sync::Arc;
 use bytes::{Bytes, BytesMut};
 use metrics::Key;
 
+use restate_encoding::ArcedSlice;
+
 use crate::identifiers::PartitionId;
 use crate::logs::{HasRecordKeys, Keys};
 use crate::net::partition_processor::PartitionLeaderService;
-use crate::net::{RpcRequest, bilrost_wire_codec, default_wire_codec, define_rpc};
+use crate::net::{RpcRequest, bilrost_wire_codec, define_rpc};
 use crate::storage::{StorageCodec, StorageEncode};
 
-#[derive(Debug, Eq, PartialEq, Clone, serde::Serialize, serde::Deserialize)]
+#[derive(Debug, Eq, PartialEq, Clone, bilrost::Message)]
 pub struct IngestRecord {
+    #[bilrost(1)]
```

Comment:
Optional: I started to use the format `#[bilrost(tag(1))]` to make it consistent with the other cases when `tag` is required.